working-groups/wg-tidb-distributions.md

```diff
@@ -0,0 +1,32 @@
+# TiDB Distributions Working Group
+
+## Goals
+
+This working group aims to build distribution packages of TiDB for different platforms and maintain them for the long-term supports.
+
+## Meetings
+
+* Not needed.
```

Comment:
why?

Reply:
We've only 2 members close to each other at that time...

Now the team is bigger, we will arrange a meeting time, thanks.